packages/plugins/MaskBox/src/SNSAdaptor/components/DrawDialog.tsx

```diff
@@ -1,27 +1,25 @@
 /* eslint-disable no-nested-ternary */
-import { useCallback } from 'react'
-import { useContainer } from 'unstated-next'
-import { makeStyles, ActionButton } from '@masknet/theme'
-import { Add, Remove } from '@mui/icons-material'
-import { useChainContext, useProviderDescriptor } from '@masknet/web3-hooks-base'
 import {
+    EthereumERC20TokenApprovedBoundary,
     FormattedAddress,
     FormattedBalance,
     ImageIcon,
     InjectedDialog,
-    PluginWalletStatusBar,
-    WalletConnectedBoundary,
     TokenPrice,
-    EthereumERC20TokenApprovedBoundary,
-    GasSettingBar,
 } from '@masknet/shared'
-import { Box, Button, DialogActions, DialogContent, TextField, Typography } from '@mui/material'
-import { formatEthereumAddress, SchemaType, useMaskBoxConstants } from '@masknet/web3-shared-evm'
-import { formatBalance, multipliedBy } from '@masknet/web3-shared-base'
 import type { NetworkPluginID } from '@masknet/shared-base'
-import type { BoxInfo } from '../../type.js'
+import { ActionButton, makeStyles } from '@masknet/theme'
+import { useChainContext, useProviderDescriptor } from '@masknet/web3-hooks-base'
+import { formatBalance, multipliedBy } from '@masknet/web3-shared-base'
+import { SchemaType, formatEthereumAddress, useMaskBoxConstants } from '@masknet/web3-shared-evm'
+import { Add, Remove } from '@mui/icons-material'
+import { Box, Button, DialogActions, DialogContent, TextField, Typography } from '@mui/material'
+import { useCallback } from 'react'
+import { useContainer } from 'unstated-next'
 import { Context } from '../../hooks/useContext.js'
 import { useI18N } from '../../locales/index.js'
+import type { BoxInfo } from '../../type.js'
+import { GasSettingBar, PluginWalletStatusBar, WalletConnectedBoundary } from '@masknet/plugin-wallet'
```

Comment:
`PluginWalletStatusBar` and `WalletConnectedBoundary` should stay in `@masket/shared`.